design-proposals/migration-target.md

```diff
@@ -0,0 +1,177 @@
+# Overview
+We are getting asks from multiple cluster admin that would like to explicitly specify the "destination" of the VM when doing Live migration.
+While this may be less important in a cloud-native environment,
+we get this ask from many users coming from other virtualization solutions, where this is a common practice.
+The same result can already be achieved today with a few steps, this is only about simplifying it with a single direct API on the single `VirtualMachineInstanceMigration` without the need to alter a VM spec.
+
+## Motivation
+In the ideal cloud native design, the scheduler is supposed to be always able to correctly identify
+the best node to run a pod (so the target pod for the VMI after the live-migration) on.
+In the real world, we still see specific use cases where the flexibility do explicitly and directly define the target node for a live migration is a relevant nice-to-have:
+- Experienced admins are used to control where their critical workloads are move to 
+- Workload balancing solution doesn't always work as expected
+- Troubleshooting a node
+- Validating a new node migrating there a specific VM
```

Comment:
> I, as a battle tested admin, ordered a new node, beause the old one broken, got the hw team to install it in the dc, ask the net team to wire it up, installed it myself, and included it in the cluster - tainted.
I drink a coffe, as this was a lot of work.
Now I want to move one of my own VMs over to it to do some sanity testing.
No other Vm on my 50 node cluster should be impacted. I just want my very own vm to move over and see if all behaves well.
I've seen enough issues with misbehaving hardware, misbheaving live migrations, cpu firmware issues, broken storage etc. this is my sanity test